### PR TITLE
fix(docs): zustand v3 context migration doc

### PR DESCRIPTION
**Pull request by dai-shi**
## Related Issues or Discussions

Fixes https://github.com/pmndrs/zustand/discussions/1276#discussioncomment-6953472

## Summary

The migration doc was misguiding.

## Check List

- [x] `yarn run prettier` for formatting code and docs

### CODE REVIEW

File: docs/previous-versions/zustand-v3-create-context.md [hunk 1]
```diff
@@ -107,25 +107,38 @@ export default function App({ initialBears }) {
...
+  const store = useContext(StoreContext)
+  if (!store) {
+    throw new Error('Missing StoreProvider')
+  }
+  return useStore(store, selector)
+}
+```
 
-+ export const Provider = ({ children }) => <MyContext.Provider value={store}>{children}</MyContext.Provider>;
+Or, reach out some third-party libraries that provide Zustand v3-like APIs:
```
Comment (sewera) [inline docs/previous-versions/zustand-v3-create-context.md:141]:
```suggestion
Or reach out to some third-party libraries that provide Zustand v3-like APIs:
```

File: docs/previous-versions/zustand-v3-create-context.md [hunk 2]
```diff
@@ -107,25 +107,38 @@ export default function App({ initialBears }) {
...
 
-+ export const useMyStore = (selector) => useStore(useContext(MyContext), selector);
-```
+- https://github.com/charkour/zustand-di
```
Comment (sewera) [inline docs/previous-versions/zustand-v3-create-context.md:143]:
```suggestion
- <https://github.com/charkour/zustand-di>
```

File: docs/previous-versions/zustand-v3-create-context.md [hunk 3]
```diff
@@ -107,25 +107,38 @@ export default function App({ initialBears }) {
...
+- https://github.com/arvinxx/zustand-utils
```
Comment (sewera) [inline docs/previous-versions/zustand-v3-create-context.md:144]:
```suggestion
- <https://github.com/arvinxx/zustand-utils>
```

File: docs/previous-versions/zustand-v3-create-context.md (hunk 2 above)
Comment (sewera) [reply docs/previous-versions/zustand-v3-create-context.md:143]: These angle brackets are the standard way of creating hyperlinks: <https://github.github.com/gfm/#autolinks>

**Review by sewera — CHANGES_REQUESTED** [review]

**Comment by dai-shi** [comment]
@sewera Thanks!!

**Review by mayank1513 — APPROVED** [review]

**Review by sewera — APPROVED** [review]

**Pull request merged**